wrappers/IWearActuators/src/IWearActuatorsWrapper.cpp

```diff
@@ -54,11 +57,30 @@ void IWearActuatorsWrapper::run()
 
 bool IWearActuatorsWrapper::open(yarp::os::Searchable& config)
 {
-    if (!config.check("actuatorCommandInputPortName") || !config.find("actuatorCommandInputPortName").isString()) {
-        yError() << LogPrefix << "actuatorCommandInputPortName parameter not found";
+    if (!config.check("actuatorCommandInputPortName") || !config.check("gloveActuatorCommandInputPortName"))
+    {
+        yError() << LogPrefix << "No actuator command input ports found.";
         return false;
     }
 
+    // Find and configure yarp port related to the actuator commands
+    if (!config.find("actuatorCommandInputPortName").isString()) {
+        yWarning() << LogPrefix << "actuatorCommandInputPortName parameter not found. Haptic feedback will not be available.";
+    }
```

Comment:
The port is still opened, isn't it?

Reply:
Yes, fixed it with https://github.com/robotology/wearables/pull/204/commits/a6456185b90e688655dbdf331a7362e845ddc6e2